### DISCUSSION TITLE
How to properly launch DisplayCAL-py3

### DISCUSSION THREAD

**Discussion by RayTracedObserver** [eoyilmaz/displaycal-py3#361] [Q&A]
So I built, launched and calibrated my display using the instructions. It worked marvelously well on Apple silicon Mac, this is awesome ! 

However I now need to calibrate another display, and I am not sure how to relaunch the already built app. I tried using make launch again while after doing cd ./displaycal-py3/ in the terminal but it didn't launch the app...

So how do I launch it ?

Thanks :) And thank you to all the contributors to the project, this revival was needed.

**Comment by RayTracedObserver** [comment]
Never mind, for some reason it worked after trying again... Launched and ready to go.

I do have one question though. Would it be possible to make a launcher that one could add in the Mac's app folder ?

**Comment by jwn2** [comment]
I'm pretty clueless about Python, so be patient with me.

On MacOS Sonoma 14.5 I installed python3, then attempted to build DisplayCAL.
The build appeared to succeed, but there is no DisplayCAL app that I can find on Mac Studio. I'm probably missing something pretty basic.

I tried:

% python3 DisplayCAL
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/Users/_[elided...]_/DisplayCal/DisplayCAL-3.9.12/DisplayCAL/__main__.py", line 4, in <module>
    from DisplayCAL.main import main
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/DisplayCAL/main.py", line 27, in <module>
    raise RuntimeError(
RuntimeError: Need Python version >= 3.8 <= 3.11, got 3.12.4

which surely isn't right.

No doubt, you have questions for information I haven't provided. Please let me know if you have any suggestions, or guidance.

Thanks!

**Reply by RayTracedObserver** [reply]
Hi, try opening the terminal, then running the following commands :



```
cd ./displaycal-py3/
make launch
```

I also have no clue about python... so I don't know what is going on, but the RunTimeError you are getting, maybe it is due to the version of python that is installed on your Mac being incompatible with this version of DisplayCAL ? If so, maybe there is a way to downgrade python to an older version ? I have Python 3.12.3

Run this command to check installed python :

`python3 --version`

However if the app launched the first time, then it's probably not an issue with the python version.
